src/app/projects/pages/autofill-project-form/autofill-project-form.component.ts

```diff
@@ -140,8 +159,57 @@ export class AutofillProjectFormComponent implements OnInit {
     this.router.navigate(['/projects', 'register'], {queryParams: params});
   }
 
-  openDialog() {
-    const dialogRef = this.dialog.open(GeoAccessionDialogComponent);
-    dialogRef.disableClose = true;
+  onDoiExistence($event: string) {
+    this.hasDoi = $event == 'Yes' ? true : false;
+    this.hasGeo = this.hasDoi && (this.hasGeo != undefined || this.hasGeo) ? false : this.hasGeo;
+  }
+
+  onGEOAccessionExistence($event: string) {
+    this.hasGeo = $event == 'Yes' ? true : false;
+    this.hasDoi = this.hasGeo && (this.hasDoi != undefined || this.hasDoi) ? false : this.hasDoi;
+  }
+
+  private onUniqueGeoAccession(geoAccession) {
+    this.loaderService.display(true, 'This may take a moment. Please wait...');
+    this.brokerService.importProjectUsingGeo(geoAccession).pipe(
+      catchError(err => {
+        this.loaderService.display(true, `Sorry, we are unable to import the project yet due to error: [${err.message}]. You can still get a spreadsheet to import the project later.
```

Comment:
Why are you absorbing the error here?

Reply:
It's not absorbing the error, the error is being displayed and proceeding with the spreadsheet download.

Reply:
You are using `catchError` but I think it can be handled in the final error handler from `subscribe`

Reply:
it's because to handle an error I have to create another observable for the download spreadsheet, so I'm combining them in one stream. This is the intended flow, if the create project failed, we'll give the whole spreadsheet to the user. The error handling inside the subscription will happen if an error occurs for both import and download.